[Cache] Added Early Hints Beta content (#2448)
cloudflare/cloudflare-docs: products/cache/src/content/about/early-hints.md

---
title: Early Hints (Beta)
pcx-content-type: concept
---

# Early Hints (Beta)

> Early Hints takes advantage of “server think time” to asynchronously send instructions to the browser to begin loading resources while the origin server is compiling the full response. By sending these hints to a browser before the full response is prepared, the browser can figure out what it needs to do to load the webpage faster for the end user.

Reviewer: ```suggestion
Early Hints takes advantage of “server think time” to asynchronously send instructions to the browser to begin loading resources while the origin server is compiling the full response. By sending these hints to a browser before the full response is prepared, the browser can figure out how to load the webpage faster for the end user.
```
